tooling/build/scripts/publishing/index.ts

```diff
@@ -304,7 +304,7 @@ function generateSitemapTree(
   const children = [...existingChildren, ...danglingDirectories]
 
   // Get the page sorting order from the FolderMeta resource
-  const pageOrder = resources.find(
+  const pageOrderFromMeta = resources.find(
```

Comment:
nit: can mark this as deprecated if we have confirmed that we will stick to the original ADRs and remove FolderMeta